### DISCUSSION TITLE
How do you deobfuscate/rename method names with il2cppinspector?

### DISCUSSION THREAD

**Discussion by RealIndica** [alextusinean/tusiphobia#18] [Q&A]
I noticed that in the "tusiphobia/appdata/il2cpp-functions.h" file you have renamed versions of the methods. It's safe to assume you didn't rename them manually so what did you use to rename them so they could be easily readable? Thank you.

**Comment by alextusinean** [comment]
I simply modified il2cppinspector's source code. Unfortunately, I only did it locally and I don't have it anymore but I think you can find something on google about chaning il2cppinspector's type name patterns.
